android/app/src/main/kotlin/app/revanced/manager/flutter/MainActivity.kt

```diff
@@ -365,6 +365,9 @@ class MainActivity : FlutterActivity() {
                         "An error occurred:\n$stack"
                     )
                 }
+            } finally {
+                inFile.delete()
+                tmpDir.deleteRecursively()
```

Comment:
Isn't this done on the flutter side already?

Reply:
Certainly, pressing the back button to return to the dashboard, or installing the app and pressing the "Open" button delete entire patcher temporary directory in flutter side.
However, if the user closes the app directly, all the temp files will remain.
Therefore, I chose to delete everything except `out.apk` from the patcher temporary directory immediately after patch ended.

(Since the temporary directories are no longer inside the cacheDir, I believe the Manager is now more responsible than ever for ensuring that the temporary directories are cleared.)